backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/api/impl/HUShipmentScheduleBL.java

```diff
@@ -701,9 +704,22 @@ private void updateHUQuantitiesFromOrderLine(
 		final I_M_HU_LUTU_Configuration lutuConfiguration = //
 				deriveM_HU_LUTU_Configuration(shipmentSchedule);
 
-		final int qtyOrderedLU = //
-				lutuConfigurationFactory.calculateQtyLUForTotalQtyTUs(lutuConfiguration, qtyTU_Effective);
-		shipmentSchedule.setQtyOrdered_LU(BigDecimal.valueOf(qtyOrderedLU));
+		//
+		// extract the packing material from lutu configuration
+		final HuPackingInstructionsVersionId versionId = HuPackingInstructionsVersionId.ofRepoId(lutuConfiguration.getM_LU_HU_PI_Item().getM_HU_PI_Version_ID());
```

Comment:
how to we know that lutuConfiguration.getM_LU_HU_PI_Item() is not null at this point?

Reply:
NPE avoided, and field description extended